develop.md

```diff
@@ -1,34 +1,57 @@
 ---
-layout: page
+layout: default
 title: Develop
 permalink: /develop/
 topnav: true
 ---
 
-This page contains a range of resources to help you create a new mod or maintain an existing mod.
-
-### Getting started
-
-* If you would like to create a new mod please see the [wiki](https://fabricmc.net/wiki/start/).
-
-* Use the online [template mod generator](./template/).
-
-* An up-to date template mod can be found on [github](https://github.com/FabricMC/fabric-example-mod).
-
-* Install the [Fabric command line tools](./cli/).
-
-* If you need some extra help we have a number of mod development channels on our [Discord](https://discord.gg/v6v4pMv).
-
-### Latest Versions
+# Develop
+
+This page provides a curated selection of resources to support both new mod creation and the maintenance of existing projects. 
+
+If you require additional help, [the Fabric Discord server](https://discord.gg/v6v4pMv) offers dedicated mod development channels with resources and advice from the community.
+
+<div class="home post-content">
+<hr>
+   <section>
+      <div class="row-3">
+         <article class="column">
+            <h3>Getting Started</h3>
+            <section class="with-button-group">
+               <p class="component-body">If you want to create a new mod, you should refer to the wiki or the work in progress documentation site.</p>
+            </section>
+            <div class="button-group">
+							<a class="button secondary" href="https://fabricmc.net/wiki/start">Fabric Wiki</a>
+							<a class="button secondary warning" href="https://docs.fabricmc.net/">[WIP] Fabric Documentation</a>
```

Comment:
Id just make it a normal link, no need for the warning color and `[WIP]` tbh. The banner on the docs site will make this quite clear 👍